packages/compass-collection/src/components/workspace/workspace.tsx

```diff
@@ -69,65 +85,87 @@ function getIconGlyphForCollectionType(type) {
   }
 }
 
+type WorkspaceProps = {
+  tabs: any;
+  closeTab: (index: number) => {
+    type: string;
+    index: number;
+  };
+  createNewTab: (props: any) => any;
+  selectOrCreateTab: (props: any) => any;
+  appRegistry: AppRegistry;
+  prevTab: () => { type: string };
+  nextTab: () => { type: string };
+  moveTab: (
+    fromIndex: number,
+    toIndex: number
+  ) => {
+    type: string;
+    fromIndex: number;
+    toIndex: number;
+  };
+  selectTab: (index: number) => {
+    type: string;
+    index: number;
+  };
+  changeActiveSubTab: (
+    activeSubTab: number,
+    id: string
+  ) => {
+    type: string;
+    activeSubTab: number;
+    id: string;
+  };
+};
+
 /**
  * The collection workspace contains tabs of multiple collections.
  */
-class Workspace extends PureComponent {
-  static displayName = 'Workspace';
+class Workspace extends PureComponent<WorkspaceProps> {
+  boundHandleKeypress: (evt: any) => void;
```

Comment:
```suggestion
  boundHandleKeypress: (evt: React.KeyboardEvent) => void;
```

Reply:
```
Type '(evt: KeyboardEvent<Element>) => void' is not assignable to type 'EventListener'.
  Types of parameters 'evt' and 'evt' are incompatible.
    Type 'Event' is missing the following properties from type 'KeyboardEvent<Element>': altKey, charCode, ctrlKey, code, and 13 more.
```

Reply:
:woman_shrugging: Then let’s leave it as is :)